### DISCUSSION TITLE
Psycopg2 sending error for `CREATE TABLE IF NOT EXISTS`

### DISCUSSION THREAD

**Discussion by bab014** [psycopg/psycopg2#1436] [Q&A] [answered]
Finding it weird that with `postgres version 9.6`  when I try to run 
```sql
CREATE TABLE IF NOT EXISTS table (
    ...
)
```
psycopg2 throws an exception 
```python
psycopg2.errors.SyntaxError: syntax error at or near "NOT"
LINE 2: CREATE TABLE IF NOT EXISTS
```

Am I doing something wrong?

**Comment by dvarrazzo** [comment, marked as answer]
Probably you are using an old server version, not the one you think you are using.

This is not a psycopg issue anyway: it's a response from a server.

**Reply by bab014** [reply]
My newbieness was showing. You are correct, the issue is with the version.
